package.json

```diff
@@ -11,8 +11,8 @@
     "avn": "./bin-public/avn"
   },
   "scripts": {
-    "test": "jshint . && jscs . && istanbul cover node_modules/.bin/_mocha --report html --",
-    "test-travis": "jshint . && jscs . && istanbul cover node_modules/.bin/_mocha --report lcovonly --"
+    "test": "eslint lib && istanbul cover node_modules/.bin/_mocha --report html --",
```

Comment:
i'd recommend adding this in `"pretest": "eslint lib"`. Also, eslint should run on the entire project - ie, `eslint .` - and `.eslintignore` should be used if you need to ignore linting files.